lib/benchmark_runner.rb

```diff
@@ -80,15 +80,20 @@ def render_graph(json_path)
 
     # Checked system - error or return info if the command fails
     def check_call(command, env: {}, raise_error: true, quiet: false)
+      quiet ||= ENV['BENCHMARK_QUIET'] == '1'
```

Comment:
```suggestion
    def check_call(command, env: {}, raise_error: true, quiet: ENV['BENCHMARK_QUIET'] == '1')
```

was this not enough?

Reply:
Yes!